src/core/profile-provider/profile-provider.ts

```diff
@@ -191,26 +180,13 @@ export class ProfileProvider {
           mapVariant !== undefined
             ? `${profileId}.${providerName}.${mapVariant}@${version}`
             : `${profileId}.${providerName}@${version}`;
-        const mapSource = await fetchMapSource(
+        mapAst = await fetchMapAST(
```

Comment:
Not sure if this fallback makes sense now. It was supposed to catch problems with AST version mismatch between brain and sdk used in client app but now we are always getting the AST and I think that if we get not valid AST from bind we will also get invalid AST from direct call so I wouldn't do this call at all. 🤔

Reply:
Jakub is right, the AST will be the same. 

We will need either resume work on registry serving any [ast version](https://www.notion.so/superface/Registry-serving-any-AST-version-9bd743eae9cb4875ae342ea86b5fc548?v=0f4018588dc0418fbba106531b48fef5) or hope it will be fast transition to comscript.